internal/cmd/common/common.go

```diff
@@ -77,3 +80,11 @@ func ReadKeyBytes(key string) ([]byte, error) {
 
 	return kb, nil
 }
+
+func EvalMode() bool {
+	return os.Getenv(EvalModeKey) == "1"
+}
+
+func NotInEvalModeError() error {
+	return ErrNotInEvalMode
+}
```

Comment:
Here we don't need a func wrapper for the error, because users can't actually modify any of the error properties. You can drop this in favor of using `ErrNotInEvalMode` directly./

Reply:
Good point, I'm not sure what I was doing there hehe.